pkgs/top-level/all-packages.nix

```diff
@@ -16792,9 +16792,6 @@ with pkgs;
 
   gnome-session-ctl = callPackage ../by-name/gn/gnome-session/ctl.nix { };
 
-  # Using 43 to match Mutter used in Pantheon
-  gnustep = recurseIntoAttrs (callPackage ../desktops/gnustep { });
```

Comment:
Please add a deprecation notice for `gnustep` in `pkgs/top-level/aliases.nix`, thanks.

Reply:
Like this?

Reply:
Yeah, looks good.